CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to Patchwork
+
+Thank you for your interest in contributing! This document provides an overview of the process of contributing to Patchwork.
+
+## Are there any prerequisites?
+
+Patchwork is a somewhat complex project, and there's a few things you should have before you start contributing.
+
+1. Basic knowledge of Java
+2. Knowledge of and / or experience using [Mixins](https://github.com/SpongePowered/Mixin)
+3. Familiarity with Fabric and modding in general
+
+
+## Contribution notes
+
+We use [MinecraftForge @ d28cd0352b6b0fe86e062f29e681c3b14572c6d5](https://github.com/MinecraftForge/MinecraftForge/tree/d28cd0352b6b0fe86e062f29e681c3b14572c6d5) as our reference commit. All features should be based on this version of Forge for consistency. For you convenience, ramidzkh has [remapped](https://github.com/PatchworkMC/YarnForge/tree/04d384add800bc395f4934507721f72eb733389f) Forge from MCP to Yarn: contributors should use this remapped version of Forge instead.
+
+This will be changed once we update to 1.15.
+
+
+## What is the general flow of contribution?
+
+1. Pick a feature to implement. There are a few ways to do this:
+    * Checking the `#wishlist` channel on the Discord server
+    * Scrolling through the [issue tracker](https://github.com/PatchworkMC/patchwork-api/issues)
+    * Picking something to implement yourself
+2. Locate mods to test the feature
```

Comment:
something like "you can also make your own test mod (link to the repo) to isolate the feature, but please make sure that it is actually used in a mod"